marklogic-data-hub/src/main/resources/scaffolding/Entity.json

```diff
@@ -1,13 +1,13 @@
 {
   "info" : {
     "title" : "placeholder",
-    "version" : "0.0.1",
-    "baseUri" : "http://example.com/",
-    "description" : "An placeholder entity"
+    "version" : "1.0.0",
+    "baseUri" : "http://example.org",

```

Comment:
I thought this needs to end with a "/" or else the trigger for validating entities will fail?

Reply:
We validate using 
```
xdmp:json-validate-node($entity-def/info, xdmp:unquote('{type: "object", properties: {"baseUri":{type:"string", format:"uri"}}, required: ["baseUri"]}'), "strict")
```
It seems any string that has the following structure is a valid URI for ML
```
?://?
``` 

I changed the baseUri to "http://example.org" since that's what the AC in the ticket said https://project.marklogic.com/jira/browse/DHFPROD-4706
Should it be "http://example.org/" ?